src/constants/tour.ts

```diff
@@ -153,18 +142,16 @@ const steps: Step.StepOptions[] = [
             on: 'bottom',
         },
         when: {
-            show() {
-                click('[href="#txbuilder"]');
-            },
+            show: () => dispatch({ type: "UPDATE_LOCATION", slug: SLUG.TXBUILDER }),
         },
     },
     {
-        beforeShowPromise: () => awaitElementExistence('.TransactionOp__config'),
+        // beforeShowPromise: () => awaitElementExistence('.TransactionOp__config'),
```

Comment:
While `awaitElementExistence('.xdrInput__input__textarea')` is a dirty hackaround, I'd keep it for now.
Laboratory loads the js for Tabs async and if someone has slow internet and clicked through these "transition" steps fast, then we may end up throwing an error when shepherd queries a non-existent element.
While that is unlikely, it's also not that uncommon to happen.

Not sure if Lab updates the store when js loaded/executed. If it does we might be able to hook into that instead of querying the DOM until elements exist